engines/account/app/controllers/account/api/v1/login_controller.rb

```diff
@@ -16,7 +17,7 @@ def login
         private
 
         def login_params
-          params.permit(:email, :password)
+          params.permit(:username, :email, :password)
```

Comment:
Because it's not efficient and it's dirty. Let's stick with explicit parameters please.